### ISSUE TITLE
Displayed path to FSCK_A.XLF is wrong

### ISSUE THREAD

**Issue by lichtmetzger** [GDX-X/PFS-BatchKit-Manager#24]
```
REM __system\fsck100\FSCK_A.XLF
if not exist "%~dp0HDD-OSD\__system\fsck100\FSCK_A.XLF" (
"%~dp0BAT\Diagbox" gd 0c
echo Files "HDD-OSD\__system\osd100\FSCK_A.XLF"
```

This line:
`echo Files "HDD-OSD\__system\osd100\FSCK_A.XLF"`

needs to be this line:
`echo Files "HDD-OSD\__system\fsck100\FSCK_A.XLF"`

**Comment by GDX-X** [comment]
Thanks I will change that.

New update coming soon with new features, maybe this week

**Comment by israpps** [comment]
FSCK is not needed.

And it's usage HAS to be discouraged

The usage of FreeHdBoot FSCK is the best thing to do

**Comment by israpps** [comment]
@GDX-X i have a recommendation for you, instead of asking the user for HDD-OSD 1.10U and hash checking, would it be better to add a plugin system?

Where user feeds a 7z (wich script checks if it's valid via some system)

This would allow users to try other HDD-OSD versions

Specially useful for Protokernel users, who will probably prefer HDD-OSD 1.00J for their systems

**Comment by GDX-X** [comment]
Yes, I can always add that. with a hard disk size check to prevent the installation of older versions on hard disks larger than 137 GB,

**Comment by israpps** [comment]
I have all versions, lba 48bit ready

**Comment by lichtmetzger** [comment]
The tool also asks for OSDSYS_A.XLF and hosdsys.elf which are both the same file - one just gets renamed to the other.

This can be very confusing if you get the HDDOSD update from the SUDC4 disc (or Sonys HDD Utility Disc) and you dont know this step. 

I didnt know about it until I found this random thread on psx-place:
https://www.psx-place.com/threads/how-to-properly-combine-hdd-osd-and-freehdboot-1-966.32416/#post-281022

Maybe the script can rename the file automatically or at least tell you to do it.

**Comment by israpps** [comment]
> The tool also asks for OSDSYS_A.XLF and hosdsys.elf which are both the same file - one just gets renamed to the other.
> 
> This can be very confusing if you get the HDDOSD update from the SUDC4 disc (or Sonys HDD Utility Disc) and you don't know about it or where to get the hosdsys.elf.
> 
> I didnt know about it until I found this random thread on psx-place: https://www.psx-place.com/threads/how-to-properly-combine-hdd-osd-and-freehdboot-1-966.32416/#post-281022
> 
> Maybe the script can rename the file automatically or at least tell you to do it.

SUDC4 provides the packages with names that don't necesarily reflect the name after install

hosdsys.elf naming ensures compatibility with FreeHdBoot merging

**Comment by GDX-X** [comment]
Fixed in 1.1.2
